numpy/core/__init__.py

```diff
@@ -106,7 +106,6 @@
 
 __all__ = ['char', 'rec', 'memmap']
 __all__ += numeric.__all__
-__all__ += fromnumeric.__all__
```

Comment:
For the record, okay to remove here because it's included via `extend_all(fromnumeric)` in `numpy/core/numeric.py`.